bili.config.ts

```diff
@@ -6,12 +6,17 @@ const config: Config = {
       tsconfigOverride: {
         include: ['src', 'ethers']
       }
-    }
+    },
   },
-
+  extendConfig: (config) => ({
+    ...config,
+    externals: []
+  }),
+  bundleNodeModules: ["nats.ws"],
   input: 'src/iam-client-lib.ts',
   output: {
-    format: ['cjs', 'esm']
+    format: ['cjs', 'esm'],
+    minify: true
```

Comment:
@dwojno Is adding minification relevant to this fix or just a good thing to do in general?

Reply:
Nope, it's not relevant, but I thought that this library is being used on the browser, so the size matters 😄